api/test_liveness_mtchoun-mouh.py

```diff
@@ -0,0 +1,49 @@
+# Generated by Selenium IDE
+import pytest
+import os
+import time
+import json
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.common.action_chains import ActionChains
+from selenium.webdriver.support import expected_conditions
+from selenium.webdriver.support.wait import WebDriverWait
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.common.desired_capabilities import DesiredCapabilities
+from selenium.webdriver.chrome.options import Options
+
+class TestLiveness():
+  def setup_method(self, method):
+    chrome_options = Options()
+    chrome_options.add_argument('--headless')
+    chrome_options.add_argument('--no-sandbox')
+    chrome_options.add_argument('--disable-dev-shm-usage')
+    self.driver = webdriver.Chrome(chrome_options=chrome_options)
+    self.vars = {}
+  
+  def teardown_method(self, method):
+    self.driver.quit()
```

Comment:
```suggestion
    self.driver.quit()
    AWS_REGION = os.environ["AWS_REGION"]
    TABLE = os.environ["REGISTRATION_TABLE"]	
    dynamodb = boto3.resource('dynamodb', region_name=AWS_REGION)
    LIVENESS_USER="mongulu liveness" 
    table = dynamodb.Table(TABLE)
    response = table.delete_item(
        Key={
            'Name':LIVENESS_USER
        }
    )
```
Rajouter ici le contenu de ton fichier api/test_liveness_remove_user.py